narrative/3_1_fcci.tex

```diff
@@ -21,6 +21,7 @@ \subsection{\gls{FCCI}}
 emerged during two recent workshops focused on this specific community.  Early
 successes will be used to showcase the effort at conferences and workshops to
 promote broader participation.
+% Shall we include reference to those workshop ?
```

Comment:
I mention it explicitly in the Schedule section - we could mention it earlier here, too.